.buildkite/pipeline.full.yml

```diff
@@ -213,11 +212,9 @@ steps:
           - "--fail-fast"
           - "--tags"
           - "not @Flaky"
+          - "--resilient"
```

Comment:
Is there any way this could be the default setting in mazerunner? Otherwise there'll be a divergence in behaviour when running with BrowserStackLocal & flakes will be more likely

Reply:
I had the same thought myself and was probably waiting to see how it performs on Android before making that leap.  It's generally much better having in than not on Cocoa, but logged errors can be harder to interpret when things go wrong.  Can we see how it runs for a bit and go from there?  We might decide we don't like the option at all on Android.